docs/GettingStarted.md

```diff
@@ -259,18 +259,38 @@ The `scanner` writes a new ORT result file to `[scanner-output-path]/scan-result
 addition to the analyzer result from the input. This way belonging results are stored in the same place for traceability.
 If the input file already contained scan results they are replaced by the new scan results in the output.
 
-## 6. Generate a report
+## 6. Running the evaluator
```

Comment:
Could you please add a reference to https://github.com/heremaps/oss-review-toolkit/issues/1236 to the commit message (but none that would auto-close that issue yet, as still more pieces are missing)?

Reply:
Done.